### ISSUE TITLE
Generate swagger parameters when using without attribute

### ISSUE THREAD

**Issue by henkosch** [trenoncourt/AutoQueryable#56]
When using AutoQueryable without the attribute, you cannot generate swagger parameters, because it is based on the AutoQueryableAttribute.

https://github.com/trenoncourt/AutoQueryable/blob/db22b07d91fb9c32b55c89f408c1b11a87855bee/src/AutoQueryable.AspNetCore.Swagger/AutoQueryableOperationFilter.cs

Possible solutions:
- Allow adding AutoQueryableAttribute even when the action return type is not an IQueryable. (Currently this throws a NullReferenceException)
- Create a new separate attribute to add the swagger OperationFilter like AutoQueryableSwaggerAttribute.

```
[Route("api/[controller]")]
public class UsersController
{
    [HttpGet]
    public IActionResult Get([FromServices] myDbContext dbContext)
    {
        var resultSet = dbContext.Product.Select(p => new { ... }).AutoQueryable(queryString);
        return new OkObjectResult(new {
            foo = "",
            ....,
            Values = resultSet;
        })
    }
}
```

**Comment by trenoncourt** [comment]
Hi @henkosch,

Thanks for your contribution.
I'll add à new attribute soon for this case.

**Comment by trenoncourt** [comment]
Will be released at the same time than AQ 2.0, soon.
